addons/l10n_syscohada/wizard/account_move_reversal.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+from odoo import models
```

Comment:
```suggestion
from odoo import models
from odoo.addons.account import SYSCOHADA_LIST
```
Instead of duplicating the list of countries, why not import it directly?